### DISCUSSION TITLE
curl 7.74.0 --write-out  %{time_namelookup} is broken

### DISCUSSION THREAD

**Discussion by smiliang** [Homebrew/discussions#353] [Everyday usage] [answered]
Hello,
curl 7.74.0 --write-out  %{time_namelookup} is broken on Mac OS 10.15.6.

curl 7.74.0 (x86_64-apple-darwin19.6.0) libcurl/7.74.0 (SecureTransport) OpenSSL/1.1.1i zlib/1.2.11 brotli/1.0.9 zstd/1.4.5 libidn2/2.3.0 libssh2/1.9.0 nghttp2/1.42.0 librtmp/2.3
Release-Date: 2020-12-09

```
curl -w "dns_resolution: %{time_namelookup}, tcp_established: %{time_connect}, ssl_handshake_done: %{time_appconnect}, TTFB: %{time_starttransfer}\n" -o /dev/null -s https://bing.com
dns_resolution: 2314, tcp_established: 504265, ssl_handshake_done: 726734, TTFB: 1220722
```

curl 7.64.1 's output is normal:

```
/usr/bin/curl -w "dns_resolution: %{time_namelookup}, tcp_established: %{time_connect}, ssl_handshake_done: %{time_appconnect}, TTFB: %{time_starttransfer}\n" -o /dev/null -s https://bing.com
dns_resolution: 0.002891, tcp_established: 0.136988, ssl_handshake_done: 0.455930, TTFB: 0.593762
```

**Comment by carlocab** [comment, marked as answer]
Are you unable to reproduce the incorrect output with non-Homebrew `curl 7.74.0`?

**Reply by smiliang** [reply]
> Are you unable to reproduce the incorrect output with non-Homebrew `curl 7.74.0`?

I just tried it and it's a problem with the curl's code, sorry to bother.

**Reply by carlocab** [reply]
This appears to have been fixed in `curl 7.75.0`.
